src/js/jquery.grideditor.js

```diff
@@ -74,7 +74,14 @@ $.fn.gridEditor = function( options ) {
         
         // Copy html to sourceElement if a source textarea is given
         if (settings.source_textarea) {
-            baseElem.html($(settings.source_textarea).val());
+            var sourceHtml = $(settings.source_textarea).val();
+            if(sourceHtml.length > 0 && $('<div>'+sourceHtml+'</div>').find('.row').addBack('.row').length == 0) {
```

Comment:
`.addBack('.row')` is unneccesary i think?
```suggestion
            if(sourceHtml.length > 0 && $('<div>'+sourceHtml+'</div>').find('.row').length == 0) {
```